### ISSUE TITLE
The question of GPU memory

### ISSUE THREAD

**Issue by zhangxuchen0806** [nnnth/UFO#34]
Thanks for this nice work.

When I train the "UFO-ViT-base" of "grounding" task, use datasets Refcoco, Refcoco+, Refcocog.
**About 5.7GB of GPU memory was used, and the fine-tune method of LoRA only use 2GB.**
The "UFO-ViT-base" of "grounding" task (5.7GB) is use your origin config "configs/UFO-ViT/single_ground_base.py", we only modify the dataset path.
A similar phenomenon occurs for the "caption" task

It is so "efficiency" ? !
Is this a normal scene?

By theway, would you be willing to share some of your training logs to help researchers better analyze their experiments? It's understandable if this is difficult

Thans anyway.

**Comment by nnnth** [comment]
I think the memory usage for grounding and caption tasks with UFO-ViT-base is normal. For both grounding and caption, the image resolution is only 224x224, UFO-ViT-base has just about 100 million parameters, and we have also used gradient checkpointing, which can significantly reduce memory consumption. In fact, all experiments with ViT-base should be trainable on RTX 4090 (24GB). 
As for the training logs, I will try to find them later. Since the experimental environments have been switched several times, some of them may have been lost.

**Comment by nnnth** [comment]
I just upload the caption and ground logs to Huggingface's repository: [caption](https://huggingface.co/kanashi6/UFO/blob/main/single_caption.log) and [ground](https://huggingface.co/kanashi6/UFO/blob/main/single_ground.log). There may be some details that are different from the current code, but the general trend should be the same. Hope it will be helpful to you.

**Comment by zhangxuchen0806** [comment]
Thanks for your give. Now we are undertaking some lightweight tasks, based on your research. 
Due to the limitations of computing power, it is impossible to conduct a large number of hype-parameter adjustment tests.
So, would you be willing to provide the training logs for insseg, detection, and semseg? It's understandable if this is difficult.
This can help me analysis my experiments.

Thanks for your support and this nice work!

**Comment by zhangxuchen0806** [comment]
I find that you use multiple GPU enviroment, since I want to reproduce your results in a single GPU, if I need to use multiple times the number of training steps?
Such as you train the ufo-base-insseg in 8 4090 GPUs for 120000 steps, I at least train the 480000 steps to produce your results.
The batch size config is same.

**Comment by zhangxuchen0806** [comment]
Thank you for your provide training log, I learned a lot. 
I found that you set checkpoint and resume the train process in instance-segment log, how to achieve this function? 

<img width="1454" height="648" alt="Image" src="https://github.com/user-attachments/assets/f36fe742-6e01-4217-a9cc-91035cbac1c6" />

How can I disrupt and resume my training process. Thank your team anyway.

**Comment by nnnth** [comment]
Very glad to help you. You can use the `resume` parameter when launching the training, for example: 
```
bash ./tools/dist_train.sh {config} 4 --resume {checkpoint_path}. 
```
You can also use `--resume 'auto'`, which allows the framework to automatically find the latest checkpoint in the checkpoint_dir (determined by the content of the `last_checkpoint` file).

**Comment by zhangxuchen0806** [comment]
Thanks again, so that，I just need input my latest checkpoint path. Right?
Should I ensure the configs are same? or I can modify this when resume the train process.
It's excited to try it, thank you bro.
